src/utils/syntax-highlight.js

```diff
@@ -192,7 +192,7 @@ function duplicateMultilineNode(element) {
 
   // wrap each new line with the current element's class
   const result = getLines(element.innerHTML)
-    .reduce((all, lineText) => `${all}<span class="${className}">${lineText || '\n\n'}</span>\n`, '');
+    .reduce((all, lineText) => `${all}<span class="${className}">${lineText || ''}</span>\n`, '');
```

Comment:
Maybe we can remove this?
```suggestion
    .reduce((all, lineText) => `${all}<span class="${className}">${lineText}</span>\n`, '');
```